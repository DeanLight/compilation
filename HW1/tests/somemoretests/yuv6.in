<6		8
	6	5			7	
	 8  3 	 2	6173 636 
	 9
	 6
	 9>