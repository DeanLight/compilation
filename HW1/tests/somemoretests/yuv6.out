6 STRING hex2ascii
6 EOF 
